mars/dataframe/core.py

```diff
@@ -1499,6 +1506,13 @@ def columns(self):
     def axes(self):
         return [self.index, self.columns]
 
+    @property
+    def empty(self):
+        shape = getattr(self, '_shape')
+        if np.nan in shape:
+            raise RuntimeError("The shape is unknown until you trigger execution")
+        return shape[0] == 0
```

Comment:
if length of columns is 0, `df.empty` also returns True

Reply:
Resolved